iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -414,8 +422,9 @@ class ConversationListViewController: FullScreenViewController,
 
     private func createNewConversation(_ sendable: Sendable) {
         Task {
-            let channelId = ChannelId(type: .messaging, id: UUID().uuidString)
-            let userIDs = Set(self.userIDs)
+            let username = User.current()?.initials ?? ""
+            let channelId = ChannelId(type: .messaging, id: username+"-"+UUID().uuidString)
+            let userIDs = Set(self.members.userIDs)
```

Comment:
Some helpful debug code. Prepend the creators initials to the conversation id so it's easier to find in the dashboard.